Assets/GothicVR/Scripts/Globals/Hero.cs

```diff
@@ -0,0 +1,31 @@
+using System.Linq;
+using GVR.Caches;
+using GVR.Properties;
+using UnityEngine.SceneManagement;
+using ZenKit.Daedalus;
+
+namespace GVR.Globals
+{
+    public static class Hero
+    {
+        public static void LoadHeroVM()
+        {
+            var hero = GameData.GothicVm.InitInstance<NpcInstance>("hero");
+            GameData.GothicVm.GlobalHero = hero;
+
+            var heroProperties = new NpcProperties();
+            LookupCache.NpcCache[hero.Index] = heroProperties;
+            
+            GvrEvents.GeneralSceneLoaded.AddListener(SetHeroGameObject);
+        }
+
+        private static void SetHeroGameObject()
+        {
+            var generalScene = SceneManager.GetSceneByName(Constants.SceneGeneral);
+            if (GameData.GothicVm.GlobalHero != null)
+                LookupCache.NpcCache[GameData.GothicVm.GlobalHero.Index].go = generalScene.GetRootGameObjects()
```

Comment:
So you desperately want to have the hero in the game?

Then please let's not search for him like that. But try to use a variable where the two elements register themself.